### ISSUE TITLE
Error "File size unexpectedly exceeded ZIP64 limit" occurs when using py-wacz on a large WACZ file

### ISSUE THREAD

**Issue by whalehub** [webrecorder/wacz-format#17]
I installed [py-wacz](https://github.com/webrecorder/wacz-format/tree/master/py-wacz) in a virtualenv on Debian 10 using Python 3.7.3 today. The alias `wacz` which is used below is just shorthand for:

```
/opt/py-wacz/bin/python3 /opt/py-wacz/bin/warcz
```

The reference command, when used on a 2.6GB WACZ file, returns the following error message for me:

```
whalehub@pdh:/srv/www/archive# wacz -o test-large.wacz test-large.warc
Generating indexes...
Writing archives...
Traceback (most recent call last):
  File "/opt/py-wacz/bin/wacz", line 33, in <module>
    sys.exit(load_entry_point('wacz==0.1.0', 'console_scripts', 'wacz')())
  File "/opt/py-wacz/lib/python3.7/site-packages/wacz-0.1.0-py3.7.egg/wacz/main.py", line 33, in main
  File "/opt/py-wacz/lib/python3.7/site-packages/wacz-0.1.0-py3.7.egg/wacz/main.py", line 78, in create_wacz
  File "/usr/lib/python3.7/zipfile.py", line 1126, in close
    raise RuntimeError('File size unexpectedly exceeded ZIP64 '
RuntimeError: File size unexpectedly exceeded ZIP64 limit
Exception ignored in: <function ZipFile.__del__ at 0x7f76611e9b70>
Traceback (most recent call last):
  File "/usr/lib/python3.7/zipfile.py", line 1789, in __del__
    self.close()
  File "/usr/lib/python3.7/zipfile.py", line 1798, in close
    raise ValueError("Can't close the ZIP file while there is "
ValueError: Can't close the ZIP file while there is an open writing handle on it. Close the writing handle before closing the zip.
whalehub@pdh:/srv/www/archive#
```

I've confirmed that the same command works fine when used on a ~30MB WACZ file instead:

```
whalehub@pdh:/srv/www/archive# wacz -o test-small.wacz test-small.warc
Generating indexes...
Writing archives...
Generating metadata...
whalehub@pdh:/srv/www/archive# ls -l
-rw-r--r-- 1 whalehub  whalehub  29078146 Sep  9 05:17 test-small.wacz
-rw-r--r-- 1 whalehub  whalehub  29071676 Sep  9 04:51 test-small.warc
whalehub@pdh:/srv/www/archive#
```

**Comment by ikreymer** [comment]
py-wacz is still in development, hope to have an alpha version soon
Thanks for testing this early version out! It looks like it was missing the file size, and did not switch to zip64.
I think it should work with the latest

**Comment by whalehub** [comment]
@ikreymer Thanks for the quick fix! I can confirm that it works now using b0e5424.
